.github/workflows/test.yml

```diff
@@ -32,6 +32,12 @@ jobs:
     - name: Check out the source code
       uses: actions/checkout@v2
 
-    - name: Test Byte Compilation
+    - name: Test Byte Compilation Against melpa stable
       run: |
-        eldev -dtT -C compile --warnings-as-errors
+        eldev -dtT -C --stable compile --warnings-as-errors
+
+    - name: Test Byte Compilation Against melpa unstable
```

Comment:
Is there a rationale for the weird camelization going on here? Why not `Test byte compilation against Melpa unstable`?

Reply:
Nope :smile: